### DISCUSSION TITLE
Scans not Running / Jobs not being created - AWS EKS on Fargate

### DISCUSSION THREAD

**Discussion by markdand** [secureCodeBox/secureCodeBox#2520] [Q&A] [answered]
Hi!

I'm running into an issue where, after deploying SecureCodeBox onto AWS EKS on Fargate, scanners don't seem to be working. I've followed the installation guide from the documentation, and am using the Nikto scanner as a test. When I query the jobs using 'kubectl get jobs', nothing ever appears.

Has anyone deployed to EKS Fargate and can tell me what I'm missing?

Thanks!

**Comment by J12934** [comment, marked as answer]
Can you take a look at the logs from the secureCodeBox operator?
It should log an error if it's unable to create the jobs

**Reply by markdand** [reply]
Hi @J12934,

Thanks for pointing me in the right direction, the error that is showing currently is 'Presigned URLs cannot be generated with anonymous credentials'. Will see if I can figure out why that error is showing.
